target/linux/ath79/nand/base-files/lib/upgrade/glinet.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0
+#
+# Copyright (C) 2019 Jeff Kletsky
+#
+
+glinet_using_boot_dev_switch() {
+	if [ "$(fw_printenv -n boot_dev 2>/dev/null)" = "on" ] ; then
+		>&2 echo "NOTE: boot_dev=on; use switch to control boot partition"
+		true
+	else
+		false
+	fi
+}
+
+glinet_set_next_boot_nand() {
+	mkdir -p /var/lock
+	! glinet_using_boot_dev_switch &&	\
+		fw_setenv bootcount 0 &&	\
+		>&2 echo "Next boot set for NAND"
+}
+
+glinet_set_next_boot_nor() {
+	mkdir -p /var/lock
+	! glinet_using_boot_dev_switch &&	\
+		fw_setenv bootcount 3 &&	\
```

Comment:
I'd use spaces instead of tabs here and above before the backslash.

Reply:
Considering that the whitespace here before the \ used for line continuation is for readability, the use of a tab meets shell-syntax requirements and meets the goals of human readability at the cost of a single byte, rather than multiples.

Reply:
Well, the readability goal is achieved by the indent, and my comment was actually for cosmetic reasons, as I consider the spaces version better for readability. However, I think you overlooked the "I'd" at the beginning of my sentence.

Reply:
"I'd" takes on a subtle difference in connotation when it comes from an OpenWrt committer.

Will be changed in next push

Reply:
Just to prevent any misunderstandings here: I wanted to express that I personally would have done that differently, based on my perception of readability. This was meant as a pointer in case you just overlooked that, or did it without explicit care, as it would be easy to adjust, without testing etc.
So, if you prefer your version, this explicitly does _not_ imply that you _should_ change it, but just is meant to add an opinion to your consideration.
I tend to use "I would" and "I personally would" to imply that a change is optional, particularly when it's effectively a matter of taste or I am not sure whether something is done by intention or accident. If that's not understood this way, maybe I have to state that more clearly in the future.